### ISSUE TITLE
Docbuilder should ask before clobbering

### ISSUE THREAD

**Issue by shakna-israel** [shakna-israel/docbuilder#23]
If the outfile already exists, docbuilder should ask before clobbering it.

Perhaps ask the user to confirm clobbering, and allow them to rename it.

This should be able to be overridden by the cli, and a quiet flag.

**Comment by shakna-israel** [comment]
e.g.

```
./docbuilder.py myscript.py myscript.md
```

```
Uh oh! File exists, [o]verwrite, or rename:
```

**Comment by shakna-israel** [comment]
The `--quiet` flag should override this behaviour and simply clobber the file.

**Comment by shakna-israel** [comment]
Fixed as of 6ab5359dfffc54fc788d5dbb055aa4ca50698cb2.
